### PR TITLE
[MISC] Updating and cleaning README.md

### PR DESCRIPTION
**Pull request by deusebio**
Based on discussion with Michelle traced in [this document](https://docs.google.com/document/d/11kH7yoAtVm8t9NsJbR7z-U2MCMBB6MMC4Bw_BlUPfd4/edit#heading=h.eju7z76sxcnf)

### CODE REVIEW

**Comment by deusebio** [comment]
@michtabirao

File: README.md [hunk 1]
```diff
@@ -1,164 +1,49 @@
...
-### Replication
-#### Adding Replicas
-To add more replicas one can use the `juju add-unit` functionality i.e.
-```shell
-juju add-unit mongodb -n <num_of_replicas_to_add>
-```
-The implementation of `add-unit` allows the operator to add more than one unit, but functions internally by adding one replica at a time, as specified by the [constraints](https://www.mongodb.com/docs/manual/reference/command/replSetReconfig/#reconfiguration-can-add-or-remove-no-more-than-one-voting-member-at-a-time) of MongoDB.
+Charmed MongoDB (VM Operator) is an enhanced, open source and fully-compatible drop-in replacement for the MongoDB Community Edition with advanced MongoDB enterprise features. This operator uses the [Charmed MongoDB snap package](https://snapcraft.io/charmed-mongodb), which offers more features than the MongoDB Community version, such as backup and restores, monitoring and security features.
 
+To see the Charmed MongoDB features and releases, visit our [Release Notes page](https://github.com/canonical/mongodb-operator/releases).
```
Comment (MiaAltieri) [inline README.md:20]:
```suggestion
To see the Charmed MongoDB features and releases, visit our [Release Notes page](https://github.com/canonical/mongodb-operator/releases). Currently the charm supports:
- Replication
- Password Rotation
- User management
- TLS
- Backup & Restore
```
once COS PR lands I can add that here as well

**Review by MiaAltieri — APPROVED** [review]
This is excellent - thanks a ton @deusebio this _really_ helps me out - one small suggestion

**Review by delgod — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,164 +1,54 @@
...
-```
 
-Similarly, the primary replica is displayed as a status message in `juju status`, however one should note that this hook gets called on regular time intervals and the primary may be outdated if the status hook has not been called recently. 
-
-Further we highly suggest configuring the status hook to run frequently. In addition to reporting the primary, secondaries, and other statuses, the status hook performs self healing in the case of a network cut. To change the frequency of the update status hook do:
-```shell
-juju model-config update-status-hook-interval=<time(s/m/h)>
-```
-Note that this hook executes a read query to MongoDB. On a production level server this should be configured to occur at a frequency that doesn't overload the server with read requests. Similarly the hook should not be configured at too quick of a frequency as this can delay other hooks from running. You can read more about status hooks [here](https://juju.is/docs/sdk/update-status-event).
+## Software and releases
```
Comment (michtabirao) [inline README.md:17]: Charmed MongoDB (VM Operator) is an enhanced, open source and fully-compatible drop-in replacement for the MongoDB Community Edition with advanced MongoDB enterprise features. This operator uses the [Charmed MongoDB snap package](https://snapcraft.io/charmed-mongodb), which offers more features than the MongoDB Community version, such as backup and restores, monitoring and security features.

To see the Charmed MongoDB features and releases, visit our [Github Release](https://github.com/canonical/mongodb-operator/releases) and Features [Release Notes](https://discourse.charmhub.io/t/release-notes-charmed-mongodb-5-vm-operator/10039) page.

File: README.md [hunk 3]
```diff
@@ -1,164 +1,54 @@
...
-```shell
-juju relate mongodb application
-```
+If you find a bug in this snap or want to request a specific feature, here are the useful links:
 
-To remove a relation:
-```shell
-juju remove-relation mongodb application
-```
+* Raise issues or feature requests in [Github](https://github.com/canonical/mongodb-operator/issues)
```
Comment (michtabirao) [inline README.md:41]: Meet the community and chat with us if there are issues and feature requests in our [Mattermost Channel](https://chat.charmhub.io/charmhub/channels/data-platform).

File: README.md [hunk 4]
```diff
@@ -1,164 +1,54 @@
...
+* Meet the community and chat with us if there are issues and feature requests in our [Mattermost Channel](https://chat.charmhub.io/charmhub/channels/data-platform)
 
 ## Contributing
 
 Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines on enhancements to this charm following best practice guidelines, and [CONTRIBUTING.md](https://github.com/canonical/mongodb-operator/blob/main/CONTRIBUTING.md) for developer guidance.
 
-
 ## License
+
 The Charmed MongoDB Operator is free software, distributed under the Apache Software License, version 2.0. See [LICENSE](https://github.com/canonical/mongodb-operator/blob/main/LICENSE) for more information.
```
Comment (michtabirao) [inline README.md:52]: delete this

File: README.md [hunk 5]
```diff
@@ -1,164 +1,54 @@
...
 
 The Charmed MongoDB Operator is free software, distributed under the Apache Software License, version 2.0. It [installs/operates/depends on] [MongoDB Community Version](https://github.com/mongodb/mongo), which is licensed under the Server Side Public License (SSPL)
```
Comment (michtabirao) [inline README.md:54]: delete this

File: README.md [hunk 6]
```diff
@@ -1,164 +1,54 @@
...
+## Software and releases
 
-### Replication
-#### Adding Replicas
-To add more replicas one can use the `juju add-unit` functionality i.e.
-```shell
-juju add-unit mongodb -n <num_of_replicas_to_add>
-```
-The implementation of `add-unit` allows the operator to add more than one unit, but functions internally by adding one replica at a time, as specified by the [constraints](https://www.mongodb.com/docs/manual/reference/command/replSetReconfig/#reconfiguration-can-add-or-remove-no-more-than-one-voting-member-at-a-time) of MongoDB.
+Charmed MongoDB (VM Operator) is an enhanced, open source and fully-compatible drop-in replacement for the MongoDB Community Edition with advanced MongoDB enterprise features. This operator uses the [Charmed MongoDB snap package](https://snapcraft.io/charmed-mongodb), which offers more features than the MongoDB Community version, such as backup and restores, monitoring and security features.
```
Comment (michtabirao) [inline README.md:18]: delete this

File: README.md [hunk 7]
```diff
@@ -1,164 +1,54 @@
...
 
+To see the Charmed MongoDB features and releases, visit our [Release Notes page](https://github.com/canonical/mongodb-operator/releases). Currently the charm supports:
```
Comment (michtabirao) [inline README.md:20]: We can delete this
To see the Charmed MongoDB features and releases, visit our [Release Notes page](https://github.com/canonical/mongodb-operator/releases). Currently the charm supports:
- Replication
- Password Rotation
- User management
- TLS
- Backup & Restore

**Pull request merged**